src-tauri/src/lib.rs

```diff
@@ -314,6 +317,99 @@ pub fn launch_northstar(
     ))
 }
 
+pub fn get_titanfall_proton() -> Option<String> {
+    let steampath = SteamDir::locate().unwrap();
+    let configpath = steampath.path.join("config").join("config.vdf");
+
+    let vdf_text = fs::read_to_string(configpath).unwrap();
+    let config = Vdf::parse(&vdf_text).unwrap().value;
+    let obj = config.unwrap_obj();
+
+    let software = obj.get("Software")?.get(0)?.get_obj()?;
+    let valve = software.get("Valve")?.get(0)?.get_obj()?;
+    let steam = valve.get("Steam")?.get(0)?.get_obj()?;
+    let compat_tool_mapping = steam.get("CompatToolMapping")?.get(0)?.get_obj()?;
+    let tfapp = compat_tool_mapping.get("1237970")?.get(0)?.get_obj()?;
```

Comment:
Not sure about other values but I'd move the Steam app id for Titanfall2 to `constants.rs` at least, that way you can pull it in other locations (e.g. https://github.com/R2NorthstarTools/FlightCore/pull/178/files#diff-eabcebd0ae5c9b77a5247e1bdbdb88b1869fc0c932d0ef0cf2ecf9ee5221be7cR382) as well ^^